scalene/scalene_profiler.py

```diff
@@ -408,9 +411,9 @@ def get_original_lock() -> threading.Lock:
     # path for the program being profiled
     __program_path: str = ""
     # temporary directory to hold aliases to Python
-    __python_alias_dir: Any = tempfile.TemporaryDirectory(prefix="scalene")
+    __python_alias_dir: Any = tempfile.mkdtemp(prefix="scalene")
```

Comment:
Can't recall - is this still in use?

Reply:
[yes](https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp)